templates/default/src/main/liberty/config/server.xml

```diff
@@ -1,11 +1,23 @@
 <server description="Liberty server">
     <featureManager>
-        <feature>microProfile-3.3</feature>
+        <feature>servlet-4.0</feature>
+        <feature>cdi-2.0</feature>
+        <feature>jaxrs-2.1</feature>
+        <feature>jaxrsClient-2.1</feature>
+        <feature>jsonb-1.0</feature>
+        <feature>jsonp-1.1</feature>
+        <feature>mpFaultTolerance-2.1</feature>
+        <feature>mpMetrics-2.3</feature>
+        <feature>mpOpenAPI-1.1</feature>
+        <feature>mpHealth-2.2</feature>
+        <feature>mpConfig-1.4</feature>
+        <feature>mpOpenTracing-1.3</feature>
```

Comment:
That's a lot of features..was wondering if we'd get by with just:

  <feature>mpMetrics-2.3</feature>
  <feature>mpHealth-2.2</feature>
  <feature>mpConfig-1.4</feature>